.eslintrc.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
         ecmaFeatures: {
             jsx: true,
         },
-        project: ['./tsconfig.json'],
+        project: ['./tsconfig.eslint.json'],
```

Comment:
I assume you know what you're doing here, because I don't :smile:

Reply:
`tsconfig.eslint.json` includes the eslint config file as well as the src folder, otherwise it's the same. It (i.e. the linter) complained about it not being included in the project defined in `.eslintrc.js`.